submit-api/migrations/versions/e635c4f73779_rename_folder_values_submitted_documents.py

```diff
@@ -0,0 +1,57 @@
+""" Update folder values in submitted_documents table
+
+Revision ID: e635c4f73779
+Revises: 32778cbd0c4a
+Create Date: 2025-05-29 12:57:15.038180
+
+"""
+from alembic import op
+from sqlalchemy.sql import table, column
+from sqlalchemy import String
+
+
+# revision identifiers, used by Alembic.
+revision = 'e635c4f73779'
+down_revision = '32778cbd0c4a'
+branch_labels = None
+depends_on = None
+
+# Define the table and columns for the update
+submitted_documents = table(
+    'submitted_documents',
+    column('folder', String)
+)
+
+# Define constants for folder values
+FOLDER_MANAGEMENT_PLAN = 'management_plan'
+FOLDER_MANAGEMENT_PLANS = 'management_plans'
+FOLDER_SUPPORTING = 'supporting'
+FOLDER_SUPPORTING_DOCUMENTS = 'supporting_documents'
```

Comment:
I did this so the folder names in submitted_document.folder match the folder names in the S3 bucket. Now we use the same const in the frontend to pass the folder name value. Before it was 2